bin/compile

```diff
@@ -357,6 +383,29 @@ export GOPATH
 
 # GB installation
 case "${TOOL}" in
+    dep)
+        eval "$(setupGOPATH ${name})"
+
+        pkgs=${GO_INSTALL_PACKAGE_SPEC:-$(<${depTOML} tq '$.metadata.heroku["install"]')}
+        if [ -z "${pkgs}" ]; then
+            pkgs="default"
+        fi
+        warnPackageSpecOverride
+        handleDefaultPkgSpec
+
+        unset GIT_DIR # unset git dir or it will mess with goinstall
+        cd "${src}"
+
+        if [ "$(< ${depTOML} tq '$.metadata.heroku["ensure"]')" != "false" ]; then
```

Comment:
What do you think of only running `dep ensure` if the manifest declares it true?

Reply:
I meant by default we encourage people to push dependencies to the repo hence `dep ensure` is not usually needed.

Reply:
Since it's the opposite of how the other managers work (govendor / glide) I didn't really consider it.

For correctness sake I'd err on running dep ensure, AFAIK it doesn't do anything network wise if the inputs-digest of the code matches whats in the repo, so for anyone who checks in the contents of vendor/ it's quick and just some disk I/O. For those that don't do check in vendor/, or forget to add the updated deps in vendor/ then running `dep ensure` saves them from themselves (or at least tries to).